apps/osim/tests/test_models.py

```diff
@@ -361,43 +372,30 @@ def test_satisfied_conditions(self, count):
                 "states": [],  # this is not valid but OK for this test
             }
         )
-        flaw = FlawFactory.build(**flaw_properties)
-
-        if flaw.is_major_incident:
-            flaw.save(raise_validation_error=False)
-            AffectFactory(flaw=flaw)
-            FlawMetaFactory(
-                flaw=flaw,
-                type=FlawMeta.FlawMetaType.REQUIRES_SUMMARY,
-                meta_attr={"status": "-"},
-            )
-        flaw.save()
+        flaw = FlawFactory()
+        for condition in conditions:
+            if condition[:3] == "has":
+                setattr(flaw, condition[4:], "valid value")
+            else:
+                setattr(flaw, condition[4:], "")
```

Comment:
use something like `verb, attr = split(" ", maxsplit=1)`, what happens if after the verb we have more than a single word? e.g. `"has affects foo"`

Reply:
makes sense, I have replaced the spaces to underscore as framework does and now it should support multiple words